Fhi.HelseId.Web/Hpr/HprService.cs

```diff
@@ -35,18 +37,19 @@ public class HprService : IHprService
         private readonly IHPR2ServiceChannel? _serviceClient;
```

Comment:
We must remove this class and the folder "Fhi.HelseId.Web\Connected Services" as we no longer use the WCF service. Then we can also remove the package references System.ServiceModel.* from our codebase 🥳

Reply:
Removed.

HprFactory, config (HprBrukernavn, HprPassord, HprUrl), memorycache etc. is also removed.